editor/code_editor.cpp

```diff
@@ -1870,6 +1870,7 @@ CodeTextEditor::CodeTextEditor() {
 
 	text_editor = memnew(CodeEdit);
 	add_child(text_editor);
+	text_editor->notification(NOTIFICATION_THEME_CHANGED); // To prevent possible crash, theme items need to be initialized directly.
```

Comment:
You could also flush the message queue, then `NOTIFICATION_THEME_CHANGED` wouldn't need to get called twice